apps/passport-client/components/screens/ScannedTicketScreens/PodboxScannedTicketScreen/sections/PodboxInstantCheckin.tsx

```diff
@@ -0,0 +1,72 @@
+import { Spacer } from "@pcd/passport-ui";
+import { useEffect } from "react";
+import { RippleLoader } from "../../../../core/RippleLoader";
+import { ErrorContainer, StatusContainer } from "../PodboxScannedTicketScreen";
+import { PodboxTicketActionErrorSection } from "./actions/PodboxTicketErrors";
+import { useExecuteTicketAction } from "./actions/useExecuteTicketAction";
+
+export function InstantCheckin({
+  ticketId,
+  eventId
+}: {
+  ticketId: string;
+  eventId: string;
+}): JSX.Element {
+  const executor = useExecuteTicketAction({
+    eventId,
+    ticketId,
+    action: {
+      checkin: true
+    }
+  });
+
+  useEffect(() => {
+    executor.execute();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
nit: is there a reason why we're ignoring deps here?

Reply:
useExecuteTicketAction returns a new variable every time, so if this hook depends on it it will just get stuck looping forever and spamming the server with check-in requests.